modules/users/tests/server/user.server.model.tests.js

```diff
@@ -153,6 +150,8 @@ describe('User Model Unit Tests:', function () {
   });
 
   after(function (done) {
-    User.remove().exec(done);
+    User.remove().exec(function() {
```

Comment:
not consistently unfortunately :(
I don't mind returning to the original `.exec(done)` and let's keep an eye really close on tests, there is a lot more work required there to get the testing automation safety-net to be really bullet proof.

Ok with you?